keps/sig-node/127-user-namespaces/README.md

```diff
@@ -603,6 +641,15 @@ use container runtime versions that have the needed changes.
 
 ### Upgrade / Downgrade Strategy
 
+Existing pods will still work as intended, as the new field is missing there.
+
+Upgrade will not change any current behaviors.
+
+When the new functionality wasn't yet used, downgrade will not be affected.
```

Comment:
I would add what happens if the functionality was used. Something like:

On downgrade, when the functionality was used, the pods created with user namespaces that are running will continue to run with user namespaces. Pods will need to be re-created to stop using the user namespace.

I'm not sure what will happen if a pod is _restarted_, like it was started with userns just fine, the kubelet is restarted to disable the feature and the pod is restarted by the kubelet. Will it use user namespaces or not? :thinking: 

I think we should mention that case too. or is it obvious to you what will happen?